crates/ingress-rpc/src/service.rs

```diff
@@ -14,16 +18,12 @@ use tips_audit::BundleEvent;
 use tips_core::types::ParsedBundle;
 use tips_core::{
     AcceptedBundle, Bundle, BundleExtensions, BundleHash, CancelBundle, MeterBundleResponse,
+    user_ops_types::{SendUserOperationResponse, UserOperationRequest},
 };
 use tokio::sync::{broadcast, mpsc};
 use tokio::time::{Duration, Instant, timeout};
 use tracing::{info, warn};
 
-use crate::metrics::{Metrics, record_histogram};
-use crate::queue::QueuePublisher;
-use crate::validation::validate_bundle;
-use crate::{Config, TxSubmissionMethod};
```

Comment:
qq: was this diff from cargo fmt? could we leave it here to reduce the diffs, if possible?